content/reference-transaction-format.md

```diff
@@ -557,6 +561,8 @@ See also: [Partial Payments](concept-partial-payments.html)
 
 These codes indicate an error in the local server processing the transaction; it is possible that another server with a different configuration or load level could process the transaction successfully. They have numerical values in the range -399 to -300. The exact code for any given error is subject to change, so don't rely on it.
 
+**Caution:** Transactions with `tel` codes are not applied to ledgers and have no effects. However, a transaction that provisionally failed may still succeed or fail with a different code after being reapplied. For more information, see [Finality of Results](#finality-of-results) and [Reliable Transaction Submission](tutorial-reliable-transaction-submission.html).
```

Comment:
nit pick:
> ... and have no effects.

They have no effects on what?

Is it more accurate to say that they do not take effect? Suggest:
> **Caution:** Transactions with `tel` codes are not applied to ledgers and do not take effect.

Or, one may prefer:
> **Caution:** Transactions with `tel` codes are not applied to ledgers and do not effect any changes to it.

Reply:
I like "cannot cause any changes to the XRP Ledger state" and think it might even be improved by terseness:
> cannot cause any changes to the XRP Ledger.